util.py

```diff
@@ -0,0 +1,10 @@
+import datetime
+
+
+def get_current_year():
+    current_date = datetime.date.today()
+    current_year = current_date.year
+    if current_date.month >= 9:  # Assuming academic year starts in September
```

Comment:
This must be discussed with @luciledierckx but it's ok for me. Maybe we want the current year to start at an arbitrary moment.

Reply:
I asked the question